creator-node/src/routes/audiusUsers.js

```diff
@@ -176,7 +176,8 @@ module.exports = function (app) {
 
         await transaction.commit()
 
-        await issueAndWaitForSecondarySyncRequests(req)
+        // Discovery only indexes metadata and not files, so we eagerly replicate data but don't await it
```

Comment:
thoughts on removing this non essential case entirely or commenting it out? thinking is we want these manual syncs to be as quick and uninterrupted as possible, cutting out all non essential jobs will help keep the queues more clear

Reply:
seems reasonable, only concern is added latency by not syncing early, bc now the final sync that is awaited has to cover a larger clock diff - in practice that would only really make a difference on heavy route like track upload so its probably fine? this is def worth adding a metric to measure duration of this particular sync request

good callout dheeraj

Reply:
i actually thought about this more and we should keep this code in there. metadata syncs get issued immediately but if there's no other write after a route is called, the next time a sync could be triggered is recurring sync. we should definitely keep this in here

Reply:
also i find it hilarious our await's were inverted based on what we needed

Reply:
I see your reason for that and think it makes sense as long as we're okay with other data not being synced until a recurring job picks it up from state machine. alternatively, we could have another queue for these lower-priority manual syncs or add them to the recurring sync queue. which route do you think is preferable here?

Reply:
yeah i think i was chatting with @SidSethi about this. i think your first instinct to keep the call without the await is the right one. the primary should make a request to the secondary to sync, doesn't need to block for a response